src/store/auth/reducer.js

```diff
@@ -0,0 +1,18 @@
+import { LOGIN_LOADING, LOGIN_ERROR, LOGIN_SUCCESS } from './types'
+
+const initState = {
+  loading: false,
+  errorTxt: null,
+  sessionID: localStorage.getItem('session_id')
+}
+
+const authReducer = (state = initState, action) => {
+  switch (action.type) {
+    case LOGIN_LOADING: return { ...state, loading: action.loading }
+    case LOGIN_ERROR: return { ...state, error: action.error, errorTxt: action.errorTxt }
```

Comment:
it looks like `error` is absent in `LOGIN_ERROR` action payload. do we really need this property in reducer?